main.js

```diff
@@ -51,65 +50,101 @@ function generateEasyNumbers(numbersWanted) {
 function generateNegativeNumbers(numbersWanted) {
```

Comment:
this function is identical to the `generateEasyNumbers` function except for the maths bit. could we just have 1 function that has TRUE/FALSE param for generating negatives or not?